src/coreclr/clrdefinitions.cmake

```diff
@@ -119,8 +119,8 @@ if(CLR_CMAKE_TARGET_LINUX)
     add_definitions(-DFEATURE_EVENTSOURCE_XPLAT)
 endif(CLR_CMAKE_TARGET_LINUX)
 # NetBSD doesn't implement this feature
-if(NOT CLR_CMAKE_TARGET_NETBSD)
-    add_definitions(-DFEATURE_HIJACK)
+if(NOT CLR_CMAKE_TARGET_NETBSD AND NOT CLR_CMAKE_TARGET_ARCH_WASM)
+    add_definitions(-DFEATURE_HIJACK AND NOT CLR_CMAKE_TARGET_ARCH_WASM)
```

Comment:
This line shouldn't have been changed, it looks like a paste issue

Reply:
yeah, last minute change with paste typo